dartagnan/src/main/java/com/dat3m/dartagnan/program/event/EventFactory.java

```diff
@@ -466,6 +476,16 @@ public static RMW newRMW(IExpr address, Register register, IExpr value, String m
     public static class Power {
         private Power() {}
 
+        public static RMWStoreExclusive newRMWStoreExclusive(IExpr address, ExprInterface value, String mo, boolean isStrong) {
+            RMWStoreExclusive store = new RMWStoreExclusive(address, value, mo, isStrong);
+            store.addFilters(Tag.RISCV.STCOND, Tag.MATCHADDRESS);
```

Comment:
Power should surely not add the `RISCV.STCOND` tag.
Also, we might want to call it `newRMWStoreConditional` in the factory to match the naming scheme of Power and RISCV, even if it creates an StoreExclusive event internally. But I'm also fine with keeping the naming scheme as is.